src/ToDo.UI/Views/HomePage.xaml

```diff
@@ -68,67 +69,68 @@
 			<muxc:NavigationView.PaneHeader>
 				<Grid Grid.Row="0"
 					  Padding="16,8"
-					  ColumnSpacing="8">
-					<Grid.ColumnDefinitions>
-						<ColumnDefinition Width="Auto" />
-						<ColumnDefinition Width="*" />
-						<ColumnDefinition Width="Auto" />
-					</Grid.ColumnDefinitions>
-
-					<PersonPicture Grid.Column="0"
-								   DisplayName="{Binding CurrentUser.Name}"
-								   d:ProfilePicture="{Binding CurrentUser.AvatarUrl}"
-								   Width="40"
-								   Height="40"
-								   VerticalAlignment="Center" />
-
-					<TextBlock Grid.Column="1"
-							   Text="{Binding CurrentUser.Name}"
-							   VerticalAlignment="Center" />
-
-					<!-- This is a hack to allow both picture and textblock to respond to tap event -->
-					<Border Background="Transparent"
+					  ColumnSpacing="8"
+					  ColumnDefinitions="*,AUto">
+					
+					<Button Grid.Column="0"
 							uen:Navigation.Request="Settings"
-							Grid.ColumnSpan="2" />
+							BorderThickness="0"
+							HorizontalAlignment="Stretch"
+							HorizontalContentAlignment="Stretch"
+							Style="{StaticResource MaterialOutlinedButtonStyle}">
```

Comment:
@agneszitte-nventive + @kazo0 Should we add some sort of chromeless button style that provide only on-hit effects with nothing extra. Since often it is required to create a clickable area that is the composition of multiple elements, here we needed a `PersonPicture+TextBlock` that doesnt look like a button, but behave like one, and the current design system doesnt have anything to support this (Here, I cheated with MaterialOutlinedButtonStyle with a BorderThickness of 0).

Reply:
@Xiaoy312  I think we can ask @NVLudwig / @AntonioBonillaHorus  for their opinion on what to use as a `Button` style in this case for the PersonPicture+TextOfTheNameOfTheUser at the top of the Navigation Menu.
Every time we have some doubts like that I think we can involve them as it will help for the generated code for the Figma plugin and not guessing each time what to do. (cc @carldebilly / @kazo0 )